docs/multi-stage-query/security.md

```diff
@@ -60,17 +60,31 @@ Depending on what a user is trying to do, they might also need the following per
 
 
 
-## S3
+## Permissions for durable storage
 
-The MSQ task engine can use S3 to store intermediate files when running queries. This can increase its reliability but requires certain permissions in S3.
+The MSQ task engine can use Amazon S3 or Azure Blog Storage to store intermediate files when running queries. This can increase its reliability but requires certain permissions.
```

Comment:
The usefulness of durable storage shouldn't be repeated here IMO. It should be added when pitching the durable storage, and reiterating it in permissions isn't necessary.

What we should add, however, is the general reason why we'd want the permissions. So something along the following lines might be cleaner in this section. 

WDYT?

```suggestion
The MSQ task engine can use Amazon S3 or Azure Blog Storage to store intermediate files when running queries. To upload, read, move and delete these intermediate files, MSQ requires certain permissions specific to the storage provider. These permissions are listed below  
```